src/convertToObject.js

```diff
@@ -1,12 +1,56 @@
-'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
   // write your code here
+    const separator = ';';
+    const styles = sourceString.split(separator);
+    const object = {};
+
+    for (const style of styles) {
+      const [property, value]  = style.split(':');
+
+      if (property && value) {
+        object[property.trim()] = value.trim();
+      }
+    }
+
+    return object;
 }
 
-module.exports = convertToObject;
+const complexStylesString = `
+  background-color:      #fff;
+-webkit-border-radius: 5px;
+  border-radius     : 5px;
+  border: 1px solid #e8e8e8;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  clear   : both       ;
+  cursor: pointer;
+  float: left;
+  font-family: inherit;
+      font-size: 14px;
+  font-weight: 400;
+  height: 42px;
+  line-height:    40px;
+  outline: 0;
+  padding-left    : 18px;
+  padding-right: 30px;
+  ;
+
+  ;
+  position: relative;
+
+
+  text-align: left !important;
+  -webkit-transition: all .2s ease-in-out;
+  transition: all .2s ease-in-out;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+
+
+  white-space: nowrap;
+  width: auto;
+`;
+
+
+convertToObject(complexStylesString);
```

Comment:
all tests failed because no need to replace module.exports

```suggestion
  module.exports = convertToObject;
```